changelog.yaml

```diff
@@ -51,6 +51,15 @@
 - version: NEXT
   date: TBD
   changes:
+    - type: internal
+      impact: minor
+      title: Implement retry on ImagePullBackOff 
+      description: |-
+        Steward is using Tekton to execute PipelineRuns.
+        Due to a [change in Tekton](https://github.com/tektoncd/pipeline/pull/4921) the behavior on ImagePullBackOff changed.
+        With this change we implement a retry mechanism to overcome problems with the changed Tekton behavior.
```

Comment:
```suggestion
        With this change we implement a retry mechanism to overcome problems with the changed Tekton behavior.
        The Tekton TaskRun is now created in the `waiting` phase (instead of the `preparing` phase.
        If the Tekton TaskRun fails the whole `waiting` phase is retried.
```